Balancer-Bots/Large-Pool-Balance-Change/src/finding.ts

```diff
@@ -1,19 +1,23 @@
 import { Finding, FindingSeverity, FindingType } from "forta-agent";
 
 export const createFinding = (data: any): Finding => {
-  let action, alertId;
+  let action, alertId, text;
 
   if (data.delta.isNegative()) {
     action = "exit";
     alertId = "BAL-5-1";
+    text = "decreased";
   } else {
     action = "join";
     alertId = "BAL-5-2";
+    text = "increased";
   }
 
   return Finding.from({
     name: `Large pool ${action}`,
-    description: `PoolBalanceChanged event detected with large ${action}`,
+    description: `Pool's(${data.poolId}) balance has ${text} by ${data.percentage.toString(

```

Comment:
```suggestion
    description: `Pool's (${data.poolId}) balance has ${text} by ${data.percentage.toString(
```

Adding the same comment here as well xD

Reply:
Oh, one more thing, as the "balance" refers to a token's balance of the pool, maybe this could be better phrased like this:

```
description: `Pool's (${data.poolId}) ${data.tokenSymbol} balance has ${text} by ${data.percentage.toString(
      10
    )}% after a large ${action}`,
```

Reply:
yes why not